github/examples_test.go

```diff
@@ -59,6 +59,35 @@ func ExampleRepositoriesService_List() {
 	fmt.Printf("Recently updated repositories by %q: %v", user, github.Stringify(repos))
 }
 
+func ExampleRepositoriesService_CreateFile() {
+	// In this example we're creating a new file in a repository using the
+	// Contents API. Only 1 file per commit can be managed through that API
+
+	// Note that authentication is needed here as you are performing a modification
+	// so you will need to modify the example to provide an oauth client to
+	// github.NewClient() instead of nil. See the following documentation for more
+	// information on how to authenticate with the client:
+	// https://godoc.org/github.com/google/go-github/github#hdr-Authentication
+	client := github.NewClient(nil)
+
+	ctx := context.Background()
+	fileContent := []byte("This is the content of my file\n\nand the 2nd line of it")
```

Comment:
nit: with two newlines, that is actually the 3rd line of it... so I would simply remove one of the two `\n`.